eclipse-glsp/glsp
# GLSP-1363: Bundle CLI tool

#### What it does
Bundle CLI tool with eslint
Also: Update tsconfig target to ES2019 to be in line with Theia
Fixes https://github.com/eclipse-glsp/glsp/issues/1363


#### How to test
- Build the application with `yarn`
- Start the cli and print the help with `yarn cli start help`
No error should occur and the output should look as follows
```bash
Usage: glsp [options] [command]

Options:
  -V, --version                                                output the version number
  -h, --help                                                   display help for command

Commands:
  coverageReport [options]                                     Generate a test coverage report for a glsp component
  release [options] <component> <releaseType> [customVersion]  Prepare & publish a new release for a glsp component
  checkHeaders [options] <rootDir>                             Validates the copyright year range (end year) of license header files
  updateNext|u [options] [rootDir]                             Updates all `next` dependencies in GLSP project to the latest version
  generateIndex [options] <rootDir...>                         Generate index files in a given source directory.
  help [command]                                               display help for command
  ```


#### Follow-ups



#### Changelog
